FaunaDB.Client/Client/FaunaClient.cs

```diff
@@ -33,8 +33,9 @@ public class FaunaClient
         public FaunaClient(
             string secret,
             string endpoint = "https://db.fauna.com:443",
-            TimeSpan? timeout = null)
-            : this(CreateClient(secret, endpoint, timeout))
+            TimeSpan? timeout = null,
+            System.Net.Http.HttpClient httpClient = null)
```

Comment:
could you add the import for this class in the top of the file and remove the full qualified name here?